.dockerignore

```diff
@@ -1,5 +1,4 @@
-# Ignores everything except dist/ prisma/ apps/ and yarn.lock
+# Ignores everything except dist/ prisma/ and apps/
 *
 !dist/
-!prisma/
-!yarn.lock
```

Comment:
do we not need the pnpm-lock.yaml file in docker? just curious why it wasn't replaced, I'm not as familiar with pnpm aside from basic use

Reply:
Yes, the lockfile is extremely recommended and sometimes even necessary (eg `pnpm install --frozen-lockfile`)

Reply:
so @BoscoDomingo you're saying I should not add it right?